Sources/ArcGISToolkit/Utility/CredentialInputView.swift

```diff
@@ -223,44 +223,48 @@ struct CredentialInputSheetView: View {
+                                passwordTextField
+                            case .usernamePassword:
+                                usernameTextField
+                                passwordTextField
+                            }
+                        } footer: {
+                            // Adding the buttons to the footer or the
+                            // form will push the buttons to the bottom of
+                            // the sheet when we want the buttons
+                            // below the text fields.
+                            HStack {
+                                Spacer()
+                                Button(role: .cancel) {
+                                    cancelAction.handler("", "")
+                                } label: {
+                                    Text(cancelAction.title)
+                                        .padding(.horizontal)
+                                }
+                                .buttonStyle(.bordered)
+                                Spacer()
+                                Button {
+                                    isPresented.wrappedValue = false
+                                    continueAction.handler(username, password)
+                                } label: {
+                                    Text(continueAction.title)
+                                        .padding(.horizontal)
+                                }
+                                .buttonStyle(.borderedProminent)
+                                .disabled(!isContinueEnabled)
+                                Spacer()
+                            }
+                            .padding(.top)
                         }
-                        Divider()
                     }
-                    .padding([.bottom, .horizontal])
-                    HStack {
-                        Spacer()
-                        Button(role: .cancel) {
-                            cancelAction.handler("", "")
-                        } label: {
-                            Text(cancelAction.title)
-                                .padding(.horizontal)
-                        }
-                        .buttonStyle(.bordered)
-                        Spacer()
-                        Button {
-                            isPresented.wrappedValue = false
-                            continueAction.handler(username, password)
-                        } label: {
-                            Text(continueAction.title)
-                                .padding(.horizontal)
-                        }
-                        .buttonStyle(.borderedProminent)
-                        .disabled(!isContinueEnabled)
-                        Spacer()
-                    }
-                    Spacer()
+                    .scrollContentBackground(.hidden)
                 }
-                .padding()
-                Spacer()
             }
+            .padding()
```

Comment:
On iOS, text field's background is getting merged with view's background so it looks odd. Don't see the same for visionOS. However, changing this makes it look much better. What is the impact of this change in visionOS? 

```suggestion
            .padding()
            .background(.thinMaterial)
```

![Simulator Screenshot - iPhone 16 Pro Max - 2024-09-23 at 13 56 22](https://github.com/user-attachments/assets/2da2c443-8b90-4242-96d7-992d28642b3f)

Reply:
> On iOS, text field's background is getting merged with view's background so it looks odd. 

It was like that before these changes. 

> Don't see the same for visionOS.

That is because using a `Form` it adds that background automatically for visionOS to follow [this](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/878#discussion_r1765903581) guideline.

> What is the impact of this change in visionOS?

Since the `Form` already gives the text fields a background there wouldn't be a need to change that.